sdk/search/Azure.Search.Documents/README.md

```diff
@@ -0,0 +1,357 @@
+foreach (SearchResult<SearchDocument> result in response.GetResults())
+{
+    // Print out the title and job description (we'll see below how to
+    // use C# objects to make accessing these fields much easier)
+    string title = (string)result.Document["business_title"];
+    string description = (string)result.Document["job_description"];
+    Console.WriteLine($"{title}\n{description}\n");
+}
+```
+You can paste that into a new console app,
+[install the Azure.Search.Documents package](#Install-the-package), add a
+`using Azure.Search.Documents;` statement, and then hit F5 to run.
+
+## Key concepts
+
+An **Azure Cognitive Search service** contains one or more **indexes** that
+provide persistent storage of searchable data in the form of JSON **documents**.
+_(If you're brand new to Search, you can make a very rough analogy between
+indexes and database tables.)_  The **Azure.Search.Documents client library**
+exposes operations on these resources through two main client types:
+
+- `SearchIndexClient` helps with
+   - [Searching](https://docs.microsoft.com/en-us/azure/search/search-lucene-query-architecture)
+     your indexed documents using
+     [rich queries](https://docs.microsoft.com/azure/search/search-query-overview)
+     and [powerful data shaping](https://docs.microsoft.com/azure/search/search-filters)
+   - [Autocompleting](https://docs.microsoft.com/rest/api/searchservice/autocomplete)
+     partially typed search terms based on documents in the index
+   - [Suggesting](https://docs.microsoft.com/rest/api/searchservice/suggestions)
+    the most likely matching text in documents as a user types
+   - [Adding, Updating or Deleting Documents](https://docs.microsoft.com/rest/api/searchservice/addupdate-or-delete-documents)
+     documents from an index
+
+- `SearchServiceClient` allows you to:
+  - [Create, delete, update, or configure a search index](https://docs.microsoft.com/rest/api/searchservice/index-operations)
+  - [Start indexers to automatically crawl data sources](https://docs.microsoft.com/rest/api/searchservice/indexer-operations)
+  - [Define AI powered Skillsets to transform and enrich your data](https://docs.microsoft.com/rest/api/searchservice/skillset-operations)
+  - [Declare custom synonym maps to expand or rewrite queries](https://docs.microsoft.com/rest/api/searchservice/synonym-map-operations)
+  - Most of the `SearchServiceClient` functionality is not yet available in our current preview
+
+_The `Azure.Search.Documents` client library (v11) is a brand new offering for
+.NET developers who want to use search technology in their applications.  There
+is an older, fully featured `Microsoft.Azure.Search` client library (v10) with
+many similar looking APIs, so please be careful to avoid confusion when
+exploring online resources.  A good rule of thumb is to check for the namespace
+`using Azure.Search.Documents;` when you're looking for us._
```

Comment:
Nit (just noticed this now): maybe not "us". To customers, we're all Microsoft. Maybe "...looking for v11 or newer".

Reply:
I'm going to leave it for this PR but will tweak when I update samples.